lua/bufferline/render.lua

```diff
@@ -231,13 +231,7 @@ local function render()
   end
 
   -- To prevent the expansion of the last click group
-  result = result .. '%0@BufferlineMainClickHandler@'
-
-  if layout.actual_width + 1 <= layout.buffers_width and len(items) > 0 then
-    local separatorPrefix = hl('BufferInactiveSign')
-    local separator = icons.bufferline_separator_inactive
-    result = result .. separatorPrefix .. separator
-  end
```

Comment:
:O Why is this gone?

Reply:
Maybe make the separator use TabLineFill and document how TabLineFill is used? But at that point we're starting to use TabLineFill for a very specific purpose and maybe we should just create a new group, auto-filled from TabLineFill, but with it's own purpose.

Reply:
Alright, I'll re-instate `BufferTabpageSpacer` as `BufferTabpageFill` and link it to `TabLineFill`.

Reply:
And the logic for the separator? `if layout.actual_width + 1 <= layout.buffers_width and ...`

That logic is there because if there is no buffers listed, a lone separator will appear and people using Startify will report bugs because it doesn't loog good. And the `actual_width + 1 < ...` thing is there because in the case there is exactly zero space available left, adding the separator is unnecessary and will shift the content.

Reply:
I'll re-add the logic, but just for the separator.